Jenkinsfile_nightly

```diff
@@ -81,6 +81,31 @@ withNightlyPipeline(type, product, component) {
     after('fortify-scan') {
       steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/Fortify Scan/**/*'
     }
+
+    after('mutationTest') {
+        publishHTML target: [
+                allowMissing         : true,
+                alwaysLinkToLastBuild: true,
+                keepAll              : true,
+                reportDir            : "build/reports/pitest",
+                reportFiles          : "index.html",
+                reportName           : "Mutation Test Report"
+        ]
+    }
+
+    after('fullFunctionalTest') {
+        steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/site/serenity/**/*'
+        steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/**/*'
```

Comment:
I am curious to know this. Do we need these steps? if yes then why are we not adding these steps to other test types i.e mutation test, smoke test.

Reply:
yes, we need these steps. 
First line is for serenity reports only and second line is to archive test reports at last stage, here functional test is last step in nightly job.